spaces/models/space.py

```diff
@@ -60,7 +60,7 @@ class Meta:
     def __str__(self) -> str:
         value = self.name
         if self.unit is not None:
-            value += f", {self.unit.name}"
+            value += f", {self.unit!s}"
```

Comment:
I think the !s here is redundant, as wouldn't{self.unit} just call the str() function on the unit, which is the same as !s? 🤔

Reply:
Wanted to be explicit here that I mean to use the str method, and not just the name property, as the unit str method has other info as well.